active-fedora.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "active-triples", '~> 0.11.0'
   s.add_dependency "deprecation"
   s.add_dependency "ldp", '~> 0.6.0'
+  s.add_dependency "ruby-progressbar", '~> 1.0'
```

Comment:
@jrochkind I'm wondering if the easier method is to create an ActiveFedora::ProgressBar class that we can implement. If this is something to use upstream, then I'd like to give it a full treatment.

I'm going to go ahead and merge this because its a good thing. I'll look into an adapter for progress bar:

Maybe: hydra-progressbar with :null, :stdout, :default strategies; that would at least mean it gets tested. The stdout mechanism would take the same parameters but would be far less helpful.